docs/contents/releases/1-18/1/index.md

```diff
@@ -1,6 +1,7 @@
-# EKS-D v1-18-eks-1 Version Depedencies
+# EKS-D v1-18-eks-1 Release
 
-The CRD (Custom Release Document) defines the release used in EKS-D.
+The [EKS-D v1-18-eks-1](https://distro.eks.amazonaws.com/kubernetes-1-18/kubernetes-1-18-eks-1.yaml)
+CRD (Custom Release Document) defines the release used in EKS-D.
```

Comment:
Can we call this release manifest instead of CRD, since we probably don't want to re-define it to mean something other than Custom Resource Definition in the Kubernetes context?

Reply:
I thought CRD in this context was talking about "Custom Resource Definition" and I'm not sure why it says Document here.